owen1, so it looks like it's the same error, just being handled by your send callback and the error listener